automated-e2e-test/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Anvith KS
```

Comment:
Copyright Anvith?

Reply:
@skdwriting Oh this should be SODA Authors. I had initially pushed this to separate github repository. Will make this change.

Reply:
Fixed